src/main.js

```diff
@@ -21,6 +21,44 @@ function loadLottie(element) {
   return animation;
 }
 
+function getHueFromHex(hex) {
```

Comment:
there is lot of cryptic math going on here and will get hard to maintain and understand. I suggest using a method out of the https://www.npmjs.com/package/tinycolor2 package. The whole module is just 15kb.

On the plus side, you will have input support in any format not limited to hex code only.

Example:
```
console.log(tinycolor("lime").toHsl().h); // 120
console.log(tinycolor("#123456").toHsl().h); // 210
```

cc @su-docker

Reply:
Done ✅